apps/myaccount/src/index.jsp

```diff
@@ -72,6 +75,12 @@
                     enablePKCE: true
                 }
 
+                if(isOrganizationManagementEnabled) {
+                    authConfig.endpoints = {
+                        authorizationEndpoint: getApiPath("/t/carbon.super/oauth2/authorize?ut=")
```

Comment:
Lets get the tenant prefix and super tenant from Java level.

Reply:
I created an issue to track this: https://github.com/wso2/product-is/issues/14658